workshop/exercise-5/README.md

```diff
@@ -1,73 +1,57 @@
-## Build and Deploy our Knative Application
+## Deploy vnext Version and Apply Traffic Shifting
 
-We've seen how to deploy an application from a container that already exists, so let's build the container image ourselves from source code. Using the Build & Serving components of Knative, we can build the image and push it to a private container registry on IBM Cloud, and then ultimately get a URL to access our application.
+Did you notice that the Fibonacci sequence started with 1? Most would argue that the sequence should actually start with 0. There's a vnext version of the application at the vnext branch in the github project that starts the sequence with 0 instead of 1. This container image has been built and deployed to dockerhub, and tagged as vnext. We'll deploy that as v2 of our app.
 
-### Install Kaniko Build Template
+### Deploy vnext
+1. Let's deploy vnext, again using a docker image on dockerhub. Maybe we want to slowly roll users over from our old version to the new version, or do some A/B testing of the new version to see what users like better. Let's see what the yaml for this looks like.
 
-As a part of this lab, we will use the kaniko build template for building source into a container image from a Dockerfile. Kaniko doesn't depend on a Docker engine and instead executes each command within the Dockerfile completely in userspace. This enables building container images in environments that can't easily or securely run a Docker engine, such as Kubernetes.
-
-A Knative BuildTemplate encapsulates a shareable build process with some limited parameterization capabilities.
-
-1. Install the kaniko build template to your cluster.
+    ```
+    cat fib-service2.yaml
+    ```
 
+    Expected Output:
     ```
-    kubectl apply --filename https://raw.githubusercontent.com/knative/build-templates/master/kaniko/kaniko.yaml
+    apiVersion: serving.knative.dev/v1alpha1
+    kind: Service
+    metadata:
+    name: fib-knative
+    namespace: default
+    spec:
+    release:
+        revisions: ["fib-knative-xxxxx", "@latest"]
+        rolloutPercent: 10
+        configuration:
+        revisionTemplate:
+            spec:
+            container:
+                image: docker.io/ibmcom/fib-knative:vnext
     ```
 
-2. Use kubectl to confirm you installed the kaniko build template, as well as to see some more details about it.  You'll see that this build template accepts parameters of `IMAGE` and `DOCKERFILE`.  `IMAGE` is the name of the image you will push to the container registry, and `DOCKERFILE` is the path to the Dockerfile that will be built.
+	Notice that we've added a rolloutPercent of 10. We've also added our old revision name to the revisions array. Your revision name will be different, so you will need to update this file.
```

Comment:
I'd mention it looks like `fib-knative-xxxxx` so they know exactly what to look for.

Reply:
It's mentioned in the step where they actually do the work, but I added it here as well.